src/main/webapp/reviewCategories/ReviewCategories.js

```diff
@@ -38,90 +85,118 @@ class ReviewCategories extends Component {
 
   init = () => {
     spinnerService.showAll();
-    Report.getReviewCategory().then(resp => {
+    this.setState({ isAdmin: component.isAdmin });
```

Comment:
It's better idea to update the state via function like: 
```suggestion
    this.setState(prev => { 
      prev.isAdmin = component.isAdmin;
      return prev;
     });
```